assets/blocks/course-outline/course-block/index.js

```diff
@@ -0,0 +1,25 @@
+import { __ } from '@wordpress/i18n';
+import { CourseIcon as icon } from '../../../icons';
+
+import edit from './edit';
+import save from './save';
+
+export default {
+	name: 'sensei-lms/course-outline',
```

Comment:
It's not totally related to this PR, but as it's for organizing, I'll give this suggestion. 😅 - We can also make it in another PR.

I noticed we have `blockNames` in the `data.js` (it's exported, but I didn't find where it's used), and we use the block names in many places. WDYT to create a file with constants and create constants for the block names?

Reply:
Yeah, I think we can export the names from the block config (to be moved to block.json) and use it there.